torch/_tensor_docs.py

```diff
@@ -515,6 +542,18 @@ def add_docstr_all(method, docstr):
 In-place version of :meth:`~Tensor.atan`
 """)
 
+add_docstr_all('atanh',
+               r"""
+atanh() -> Tensor
+
+See :func:`torch.atanh`
+""")
+
+add_docstr_all('atanh_',
+               r"""
+In-place version of :meth:`~Tensor.atanh`
+""")
```

Comment:
Docs for in-place versions of ops have been added. Thanks @mruberry for pointing to the right file. :)